crates/hc_signal_srv/src/lib.rs

```diff
@@ -0,0 +1,69 @@
+use clap::Parser;
```

Comment:
I think you have a merge issue here, this file shouldn't exist.

I also notice this is true for develop, maybe your non-backport PR had this same issue?